### PR TITLE
empty lines deletion

### PR DESCRIPTION
**Pull request by jeewanjotkaur**

### CODE REVIEW

File: submissions/pr3.2/about.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+Open source, Machine learning and development.
+# Contact Info
+Email: [divyam3897@gmail.com](mailto:divyam3897@gmail.com) 
+Twitter: [@divyam3897](https://twitter.com/divyam3897) 
+Facebook profile: [Divyam Madaan](https://www.facebook.com/divyam.madaan.1)
+Linkedin: [Divyam Madaan](https://www.linkedin.com/in/divyam-madaan-0bb627105/)
+Github: [divyam3897](https://github.com/divyam3897)
+# Emoji
+My favorite emoji is :smile:
+
```
Comment (divyam3897) [inline submissions/pr3.2/about.md]: Empty line?

**Review by divyam3897 — CHANGES_REQUESTED** [review]
You were not required to change the whole file, just remove the empty files and only those should be there as changes.

**Review by gurpreetsingh00885 — CHANGES_REQUESTED** [review]
You need to squash the two commits into one and write a good commit message. Rebase is required toi

**Comment by divyam3897** [comment]
Already fixed in https://github.com/pclubuiet/Hacktoberfest-2k17/commit/83595182d1750e6d43f65943d71a129c497b9f05

**Pull request closed without merging**